src/d3a/d3a_core/launcher.py

```diff
@@ -56,6 +55,7 @@ def run(self):
             self.job_array = [j for j in self.job_array if j.poll() is None]
 
     def is_crowded(self):
+        check_redis_health(redis_db=StrictRedis.from_url(REDIS_URL, retry_on_timeout=True))
```

Comment:
AFAIU, the call of the function does not have an effect (the return value "reconnect_needed" is not used. Do I miss something?

Reply:
Since the same method is used in `d3a-web` too where it is needed to know if reconnect_needed or not whereas here it would simply work as wait for redis to be available again to receive incoming jobs and no resubscription is needed in `d3a` so I think  it is safe to simply ignore `reconnect_needed` here

Reply:
I see. ok.